README.md

```diff
@@ -11,7 +11,7 @@ In fact, it is very easy to start using the Orb.
 All you need to do is:
 
 1. Follow the instructions at the [Orb Quick Start Guide](https://circleci.com/orbs/registry/orb/snyk/snyk#quick-start) to enable usage of Orbs in your project workflow.
-2. Set up an environment variable (`SNYK_TOKEN`) with your Snyk API token, which you can get from your [account](https://app.snyk.io/account).
+2. Set up an environment variable (`SNYK_TOKEN`) with your Snyk API token, which you can get from your [account](https://snyk.co/udtEr).
```

Comment:
The new link goes to the Dashboard for me. Is that intentional?

Reply:
Was just a marketing link, I'll ask them to update it

Reply:
Good catch btw